### PR TITLE
termui: add show_cursor() and hide_cursor()

### PR DESCRIPTION
**Pull request by jaddison06**
Added Context.show_cursor() and hide_cursor(), and implemented them in the inits. Basically copied from https://github.com/vlang/v/blob/38495da07e604fdaafca92b1dfbb812a1f3bdbae/vlib/term/control.v#L91-L94

### CODE REVIEW

**Review by serkonda7 — APPROVED** [review]

**Review by serkonda7 — COMMENTED** [review]
The code looks good :+1:

**Comment by jaddison06** [comment]
I haven't tested the init on Windows, but it's doing exactly what it was before, just in a different place, so I don't foresee any problems

File: vlib/term/ui/input_windows.c.v [hunk 1]
```diff
@@ -68,7 +68,7 @@ pub fn init(cfg Config) &Context {
 	}
 
 	if ctx.cfg.hide_cursor {
-		print('\x1b[?25l')
+		ctx.hide_cursor()
```
[the remark this answers is not in the document]
Comment (jaddison06) [reply vlib/term/ui/input_windows.c.v:71]: Good point - I guess print() was doing that before

**Review by spaceface777 — APPROVED** [review]
Good work.

**Pull request merged**